sycl/test-e2e/Matrix/SPVCooperativeMatrix/joint_matrix_apply_two_matrices.cpp

```diff
@@ -11,7 +11,7 @@
 // RUN: %{run} %t.out
 
 // XFAIL: cpu
-// XFAIL: gpu
+// XFAIL: gpu && !gpu-intel-dg2
```

Comment:
@MrSidims , should this xfail be removed completely? I notice it passes for me on PVC as well.